roles/fairfood-dependencies/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
     - imagemagick
     - memcached
     - nginx
-    # For the mysql2 gem version 0.3
-    - libmariadbclient-dev
+    # For the mysql2 gem
+    - libmariadb-dev
```

Comment:
I saw some references online about changing to `libmariadbclient-dev-compat` or `libmariadb-dev`.
I found that the latter works.

I did notice later that the schema dump now says "MariaDB dump" instead of "MySQL dump".
https://github.com/ceresfairfood/fairfood-install/pull/22/commits/9b1a1090043442f615cb5fffe870cd4423dd6dff#diff-28a7f21c68f5d76f2e9349a203bdf5c672079988914877cefadd1939d26fba8dL1-R1

I don't know if that's indicating I'm using MariaDB/MySQL in a different way, or if that's a problem (or if it's simply displayed differently in a later version)... do you have any clues @mkllnk ?

Reply:
That shouldn't be a problem at all. If the gem compiles and the app runs then all is good. I reckon that it's just due to newer versions of Mariadb actually revealing itself as Mariadb while old versions kept the Mysql name for compatibility.